iidm/iidm-api/src/main/java/com/powsybl/iidm/network/LoadingLimits.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+import java.util.Collection;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ * @author Thibaut Vermeulen <thibaut.vermeulen at rte-france.com>
+ */
+public interface LoadingLimits extends OperationalLimits {
+
+    /**
+     * Temporary current limit.
+     */
+    interface TemporaryLimit {
+
+        /**
+         * Get the temporary limit name
+         * @return the temporary limit name
+         */
+        String getName();
+
+        /**
+         * Get the temporary limit value.
+         * @return the temporary limit value
+         */
+        double getValue();
+
+        /**
+         * Get the acceptable duration of the limit in second.
+         * @return the acceptable duration of the limit in second
+         */
+        int getAcceptableDuration();
+
+        /**
+         * Check if the temporary limit is a real limit corresponding to an overloading  protection or just an operating
+         * rule
+         * @return false if it is a real limit, false otherwise
+         */
+        boolean isFictitious();
```

Comment:
I am really at ease with this function name choice. I imagine that we have it from the actual API. What do you think ?

Reply:
I agree: we should rename this method. This method returns true if there is an protection automaton that monitor the equipment. I think we can find a better name.

Reply:
Okay, I agree, is `hasOverloadingProtection()` an okay name?
Also, I am not sure limits other than current limits can have protection automaton, should this be specific to CurrentLimits.TL?